content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,327 @@
+---
```

Comment:
Is there a reason this is a top level file in "Prisma Schema" vs. a sub page of "Relations" where this feature is a sub functionality?

Reply:
Should it maybe even live under the "Preview Features" area (concepts/components/preview-features) as long as it is a preview feature?

Reply:
My thinking was that its adding context to the PSL so belongs under the Prisma schema section

Reply:
I would say it adds details to how Relations work in the Prisma Schema.

But despite that, currently it is a preview feature so should be clearly marked as such and also possible live, where other preview features live.

Reply:
It is clearly marked as preview here: 
![preview](https://user-images.githubusercontent.com/22930449/123788435-4a8b9b00-d8dc-11eb-8ac0-77dfab9e2833.PNG)

But I can move the whole thing to under the preview features node, maybe that is better

Reply:
For now i will creat a Relations leaf and put the current page into the index and add ref actions as a node. In a separate PR I can split it up per above